bikeshed/boilerplate/whatwg/computed-metadata-LS-COMMIT.include

```diff
@@ -0,0 +1,5 @@
+{
+  "Title": "[SPECTITLE] Standard Commit Snapshot [COMMIT-SHA]",
```

Comment:
This is a change (removing parentheses), but I guess it's a bit of an improvement.

Reply:
This was also intended, to align Review Drafts and Commit Snapshots on a single style.